spec/factories/educations.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+
+FactoryGirl.define do
+
+  factory :user do
+    email { Faker::Internet.email }
+    password { "password" }
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+  end
+
+  factory :application do
+    association :user
+    street { Faker::Address.street_address }
+    city { Faker::Address.city }
+    state { Faker::Address.state_abbr }
+    country { Faker::Address.country }
+    zip_code { Faker::Address.zip }
+    phone_number { Faker::PhoneNumber.cell_phone }
+  end
+
+  factory :education do
+    association :application
+    school { Faker::Educator.secondary_school }
+    location { Faker::Educator.campus }
+    start_date { Date.new(2001, 01, 01) }
+    end_date { Date.new(2008, 05, 30) }
+    degree { Faker::Educator.course }
+    degree_date { Date.new(2008, 06, 01) }
+
+    after(:build) do |education|
+      application = build(:application)
+      application.educations << education
+      user = build(:user)
+      user.application = application
+    end
+  end
+end
```

Comment:
So let's define each factory in its own file. When I open up the `factories/educations.rb` file, I don't expect to see the users or applications factories. If I were looking for the factories, it might take me a bit to find them because I wouldn't know to look in this file.